java/runtime/src/main/java/io/ray/runtime/context/RuntimeContextImpl.java

```diff
@@ -65,6 +66,11 @@ public List<NodeInfo> getAllNodeInfo() {
     return runtime.getGcsClient().getAllNodeInfo();
   }
 
+  @Override
+  public List<ActorInfo> getAllActorInfo() {
```

Comment:
Can we also support a API with an input ActorId? I saw it is supported in python.
```
  public List<ActorInfo> getAllActorInfo(ActorId actorId) 
```

Reply:
1. GlobalStateAccessor current have `GetActorInfo` api.
2. I prefer add a new api `GetActorInfo` in runtime context to support this commit.